src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -298,11 +321,47 @@ public override TaintedDataAbstractValue VisitObjectCreation(IObjectCreationOper
                             }
                         }
                     }
+
+                    if (isSanitizingMethod)
+                    {
+                        if (!sanitizedParameterPairs.Any())
```

Comment:
You could probably use `Count` or `Length` here instead.

Reply:
I recall previously there was a ban for LINQ in analyzers to avoid hidden allocations. Otherwise what is the benefit of using `Length` over `Any`?

Reply:
I also thought there was an analyzer to suggest `Count` instead of `Any`, I will need to look if it was disabled or if there as some special cases. `Count`/`Length` is a property access so it's faster and you don't have any allocation.